r/tests/testthat/test-dataset.R

```diff
@@ -128,6 +133,18 @@ test_that("filter() with %in%", {
   )
 })
 
+test_that("filter() on timestamp columns", {
+  ds <- open_dataset(dataset_dir)
+  expect_equivalent(
+    ds %>%
+      # Replace `0L` with a proper lubriate object.
+      filter(ts >= 0L , part == 1) %>%
```

Comment:
If I'm reading this correctly, this is a trivial filter (date greater than the epoch) and it yields every row. I'd prefer if the filter returned (for example) only the last three rows.